### ISSUE TITLE
Invalid markup for Spatial Tech hours on home page.

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.org#525]
Unordered lists can only contain list-items (`li`), not other elements such as headings. Update this markup so the page validates and accessibility tests don't fail.

Related to #217.
